test/python/providers/test_backend_v2.py

```diff
@@ -65,6 +78,29 @@ def test_transpile(self, opt_level):
             ],
         )
         self.assertTrue(Operator(tqc).equiv(qc))
+        self.assertMatchesTargetConstraints(tqc, self.backend.target)
+
+    @combine(
+        opt_level=[0, 1, 2, 3],
+        gate=["cx", "ecr", "cz"],
+        bidirectional=[True, False],
+        dsc=(
+            "Test GHZ circuit with {gate} using opt level {opt_level} on backend "
+            "with bidirectional={bidirectional}"
+        ),
+        name="{gate}_level_{opt_level}_bidirectional_{bidirectional}",
+    )
+    def test_5q_ghz(self, opt_level, gate, bidirectional):
+        backend = FakeBackend5QV2(bidirectional)
+        qc = QuantumCircuit(5)
+        qc.h(0)
+        getattr(qc, gate)(0, 1)
+        getattr(qc, gate)(1, 2)
+        getattr(qc, gate)(2, 3)
+        getattr(qc, gate)(3, 4)
```

Comment:
```suggestion
        getattr(qc, gate)(0, 1)
        getattr(qc, gate)(2, 1)
        getattr(qc, gate)(2, 3)
        getattr(qc, gate)(4, 3)
```
It might be good to have some of these be opposite the native gate direction from `FakeBackend5QV2`.